src/modules/signer/signer.service.ts

```diff
@@ -502,4 +505,43 @@ export class SignerService {
       sig
     )}`;
   }
+
+  /**
+   * Set `_isEthSigner` value based on a signed message.
+   * Generates a test message and signs it.
+   */
+  private async _setIsEthrSigner() {
```

Comment:
Why not use _calculatePubKeyAndIdentityToken?

Reply:
We can, but _calculatePubKeyAndIdentityToken is making a call to RPC asking about block number, so for performance reason i decide to chose this approach